Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/ColorPicker/ColorHexEdit.cpp

```diff
@@ -61,7 +61,7 @@ ColorHexEdit::ColorHexEdit(QWidget* parent)
     , m_red(0)
     , m_green(0)
     , m_blue(0)
-    , m_alpha(0)
+    , m_alpha(1)
```

Comment:
`qreal` is a typedef for `double`, so these definitions should be doubles:
```
    , m_red(0.0)
    , m_green(0.0)
    , m_blue(0.0)
    , m_alpha(1.0)
```

Reply:
Since I already started the AR, I'll either adjust this if I need to resubmit or save it for a future submission. 

The compiler will generate the same code for both, but I totally agree it's more pleasing to the eye to have the constants in the correct type. :)